Classes/Issues/Merge/IssueMergeButtonCell.swift

```diff
@@ -0,0 +1,89 @@
+//
+//  IssueMergeButtonCell.swift
+//  Freetime
+//
+//  Created by Ryan Nystrom on 2/11/18.
+//  Copyright © 2018 Ryan Nystrom. All rights reserved.
+//
+
+import UIKit
+import IGListKit
+import SnapKit
+
+final class IssueMergeButtonCell: IssueCommentBaseCell, ListBindable {
+
+    private let buttonBackgroundView = UIView()
+    private let mergeButton = UIButton()
+    private let optionButton = UIButton()
+    private let optionBorder = UIView()
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+
+        contentView.addSubview(buttonBackgroundView)
+        buttonBackgroundView.addSubview(mergeButton)
+        buttonBackgroundView.addSubview(optionButton)
+        buttonBackgroundView.addSubview(optionBorder)
+
+        buttonBackgroundView.layer.cornerRadius = Styles.Sizes.avatarCornerRadius
+        buttonBackgroundView.snp.makeConstraints { make in
+            make.edges.equalToSuperview().inset(UIEdgeInsets(
+                top: Styles.Sizes.rowSpacing/2,
+                left: Styles.Sizes.commentGutter,
+                bottom: Styles.Sizes.rowSpacing,
+                right: Styles.Sizes.commentGutter
+            ))
+        }
+
+        optionButton.setImage(UIImage(named: "chevron-down")?.withRenderingMode(.alwaysTemplate), for: .normal)
```

Comment:
Can we add an `accessibilityLabel` to this guy? :)

Reply:
Yes! I’m gonna refactor this whole button too. Good eye.